real_estate/models/estate_property.py

```diff
@@ -0,0 +1,33 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(string="Property Name", required=True, help="Enter the property name.")
```

Comment:
Bad Readability
```suggestion
    name = fields.Char(
        string="Property Name",
        required=True,
        help="Enter the property name."
    )
```
Please follow [PEP8 Maximum line guide](https://peps.python.org/pep-0008/#maximum-line-length)